docs/ADD_NEW_EXCHANGE.md

```diff
@@ -0,0 +1,1062 @@
+| BTCMarkets | Yes | No       | NA  |
+| BTSE | Yes | Yes | NA |
+| COINUT | Yes | Yes | NA |
+| Exmo | Yes | NA | NA |
+| NewExampleName | Yes | Yes | No | // <-------- 
+| CoinbasePro | Yes | Yes | No|
+| Coinbene | Yes | No | No |
+| GateIO | Yes | Yes | NA |
+| Gemini | Yes | Yes | No |
+| HitBTC | Yes | Yes | No |
+| Huobi.Pro | Yes | Yes | NA |
+| ItBit | Yes | NA | No |
+| Kraken | Yes | Yes | NA |
+| Lbank | Yes | No | NA |
+| LakeBTC | Yes | No | NA |
+| LocalBitcoins | Yes | NA | NA |
+| OKCoin International | Yes | Yes | No |
+| OKEX | Yes | Yes | No |
+| Poloniex | Yes | Yes | NA |
+| Yobit | Yes | NA | NA |
+| ZB.COM | Yes | Yes | NA |
+```
+
+#### exchanges\support.go:
+```go
+var Exchanges = []string{
+	"binance",
+	"bitfinex",
+	"bitflyer",
+	"bithumb",
+	"bitmex",
+	"bitstamp",
+	"bittrex",
+	"btc markets",
+	"btse",
+	"coinbasepro",
+	"coinbene",
+	"coinut",
+	"exmo",
+	"brandnewexamplename", // <--------
+	"gateio",
+	"gemini",
+	"hitbtc",
+	"huobi",
+	"itbit",
+	"kraken",
+	"lakebtc",
+	"lbank",
+	"localbitcoins",
+	"okcoin international",
+	"okex",
+	"poloniex",
+	"yobit",
+    "zb",
+```
+
+#### exchanges\exchange_test.go:
+```go
+func TestExchange_Exchanges(t *testing.T) {
+	t.Parallel()
+	x := exchangeTest.Exchanges(false)
+	y := len(x)
+	expected := 28 // modify this value to match the total count of exchanges
+	if y != expected {
+     t.Fatalf("expected %v received %v", expected , y)
+	}
+}
+```
+
+#### gctscript\wrappers\gct\exchange\exchange_test.go:
+```go
+func TestExchange_Exchanges(t *testing.T) {
+	t.Parallel()
+	x := exchangeTest.Exchanges(false)
+	y := len(x)
+	expected := 28 // modify this value to match the total count of exchanges
+	if y != expected {
+    	t.Fatalf("expected %v received %v", expected , y)
+	}
+}
+```
+
+#### cmd\documentation\exchange_templates:
+
+- Create a new file named <exchangename>.tmpl
+- Copy contents of template from another exchange example here being Exmo
+- Replace names and variables as shown:
+
+```go
+{{define "exchanges exmo" -}} // exmo -> ftx
+{{template "header" .}}
+## Exmo Exchange
+
+#### Current Features
+
++ REST Support // if websocket or fix are supported, add that in too
+```
+
+```go
+main.go
+var e exchange.IBotExchange // We name the exchange.IBotExchange variable after the first character of the exchange, eg f for FTX. e -> f
+
+for i := range bot.Exchanges {
+  if bot.Exchanges[i].GetName() == "Exmo" { // Exmo -> FTX
+    e = bot.Exchanges[i] // e -> f
+  }
+}
+
+// Public calls - wrapper functions
+
+// Fetches current ticker information
+tick, err := e.FetchTicker() // e -> f 
+if err != nil {
+  // Handle error
+}
+
+// Fetches current orderbook information
+ob, err := e.FetchOrderbook() // e -> f (do so for the rest of the functions too)
+if err != nil {
+  // Handle error
+}
+```
+
+- Run documentation.go to generate readme file for the exchange:
+```bash
+cd gocryptotrader\cmd\documentation
+go build
+go run documentation.exe
```

Comment:
If you do go build, a binary will be produced so the follow execution command would either be ./documentation or documentation.exe. Alternatively like the other examples you've done, you can specify `go run documentation.go`